content/content/Examples/stateless-app-mirror.md

```diff
@@ -0,0 +1,290 @@
+        ├── .wh.Endpoints_guestbook_redis-master.yaml
+        ├── .wh.Endpoints_guestbook_redis-slave.yaml
+        ├── .wh.EndpointSlice_guestbook_frontend-bkqbs.yaml
+        ├── .wh.EndpointSlice_guestbook_redis-master-hxr5k.yaml
+        ├── .wh.EndpointSlice_guestbook_redis-slave-8wt7z.yaml
+        ├── .wh.Pod_guestbook_frontend-5fd859dcf6-5nvbm.yaml
+        ├── .wh.Pod_guestbook_frontend-5fd859dcf6-j8w94.yaml
+        ├── .wh.Pod_guestbook_frontend-5fd859dcf6-s9x8p.yaml
+        ├── .wh.Pod_guestbook_redis-master-55d9747c6c-6f9bz.yaml
+        ├── .wh.Pod_guestbook_redis-slave-5c6b4c5b47-jnrsr.yaml
+        ├── .wh.Pod_guestbook_redis-slave-5c6b4c5b47-xz776.yaml
+        ├── .wh.ReplicaSet_guestbook_frontend-5fd859dcf6.yaml
+        ├── .wh.ReplicaSet_guestbook_redis-master-55d9747c6c.yaml
+        └── .wh.ReplicaSet_guestbook_redis-slave-5c6b4c5b47.yaml
+
+2 directories, 24 files
+```
+
+**NOTE**
+
+What `crane transform` is doing, is iterating through the configured plugins and
+running them against the exported resources from the prevoius step. You can see
+which plugins are configured with `crane transform list-plugins` **and** you can
+see optional arguments to those plugins with `crane transform optionals`.
+
+Now is a great time to explore what plugins you can configure with `crane
+plugin-manager list`, install one, and maybe customize the exported resources
+with something like:
+
+```bash
+crane transform --optional-flags="add-annotations=custom-crane-annotation=foo"
+```
+
+If the flags start to get tough to manage via the command-line, you can also
+specify a `--flags-file` that might look something like:
+
+```yaml
+debug: false
+export-dir: myExport
+transform-dir: myTransform
+output-dir: myOutput
+optional-flags:
+  add-annotations: "custom-crane-annotation=foo"
+```
+
+# Apply Transformations
+
+Crane's `apply` command takes the exported resources + transformations saved as
+JSON patches and renders the results as YAML files that you _should_ be able to
+apply to another cluster as is.
+
+```bash
+crane apply
+```
+
+**NOTE**
+
+To better understand what `crane` is doing during this step, go back and look at
+one of the transformations created in the last step:
+
+```bash
+$ cat transform/resources/guestbook/transform-Deployment_guestbook_frontend.yaml
+[{"op":"remove","path":"/metadata/uid"},{"op":"remove","path":"/metadata/resourceVersion"},{"op":"remove","path":"/metadata/creationTimestamp"},{"op":"remove","path":"/metadata/generation"},{"op":"remove","path":"/status"}]% 
+```
+
+So when crane "apply"s the transformations for the guestbook frontend, it will
+be executing a handful of JSON patches that:
+
+1. Remove the UID
+1. Remove the resourceVersion
+1. Remove the creationTimestamp
+1. Remove the generation field
+1. Remove the status
+
+All of this data is leftovers from the "source" cluster, and we are removing
+them from the final manifests to make them applicable to the "destination"
```

Comment:
Do we want to mention the manifests should now be cluster agnostic, and therefore also appropriately non-specific enough to be versioned and managed via gitops / CD?